js/zb.js

```diff
@@ -1064,18 +1064,34 @@ module.exports = class zb extends Exchange {
         return this.parseOrders (response, market, since, limit);
     }
 
-    async fetchOpenOrders (symbol = undefined, since = undefined, limit = 10, params = {}) {
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         if (symbol === undefined) {
             throw new ArgumentsRequired (this.id + 'fetchOpenOrders() requires a symbol argument');
         }
         await this.loadMarkets ();
         const market = this.market (symbol);
+        const swap = market['swap'];
         const request = {
-            'currency': market['id'],
-            'pageIndex': 1, // default pageIndex is 1
-            'pageSize': limit, // default pageSize is 10
+            'pageSize': limit, // default pageSize is 10 for spot, 30 for swap
```

Comment:
Let's always put both `since/limit` under conditionals `if (since/limit !== undefined)`, cause we should not send a parameter if the user has not specified it, and if the exchange does not require it (has a default limit).